LocationManager/INTULocationManager/INTULocationManager.h

```diff
@@ -125,6 +130,33 @@ __INTU_ASSUME_NONNULL_BEGIN
 /** Immediately cancels the location request (or subscription) with the given requestID (if it exists), without executing the original request block. */
 - (void)cancelLocationRequest:(INTULocationRequestID)requestID;
 
+#pragma mark Heading Requests
+
+/** 
+ Whether or not the CLLocationManager is currently sending heading updates. 
+ */
+@property (nonatomic, assign, readonly) BOOL isUpdatingHeading;
+
+/**
+ * Specifies the minimum amount of change in degrees needed for a heading service update. Observers will not be notified of updates less than the stated filter value.
+ *
+ * @discussion The default value for this is 1.
+ */
+@property (nonatomic) INTUHeadingFilterAccuracy headingFilter;
```

Comment:
I think that would work for both location and heading requests. Distance & heading filters don't appear to do anything to improve battery life (just control when observers are notified) so it's probably not important enough to expose through its own initializer. I'll just remove the `headingFilter` from this PR to at least get this merged in and then open a new PR with the `headingFilter` work and another for `distanceFilter` once I have more time to do them that way.